gateway/src/apicast/proxy.lua

```diff
@@ -263,16 +289,22 @@ function _M:rewrite(service)
   local ctx = ngx.ctx
   local var = ngx.var
 
+  local parsed_usage = parse_usage(usage)
+
   -- save those tables in context so they can be used in the backend client
-  ctx.usage = usage_params
+  ctx.usage = parsed_usage
   ctx.credentials = credentials
-  ctx.matched_patterns = matched_patterns
 
   self.credentials = credentials
-  self.usage = usage_params
+  self.usage = parsed_usage
 
   var.cached_key = concat(cached_key, ':')
 
+  if debug_header_enabled then
```

Comment:
This is always true because `debug_header_enabled` is a function.

Reply:
Right. Forgot the `()`.